iOS/Flows/Conversation/Conversation List/ConversationListViewController.swift

```diff
@@ -374,26 +349,35 @@ extension ConversationListViewController: MessageSendingViewControllerType {
                                                                          extraData: [:])
 
                 try await controller.synchronize()
+
+                ConversationsManager.shared.activeConversation = controller.conversation
+
                 try await controller.createNewMessage(with: sendable)
             } catch {
-                logDebug(error)
+                logError(error)
             }
         }.add(to: self.taskPool)
     }
 
     func sendMessage(_ message: Sendable) {
-        guard let cid = self.getCurrentMessageSequence()?.streamCID else { return }
+        guard let cid = self.getCurrentMessageSequence()?.streamCID else {
+            self.createNewConversation(message)
+            return
```

Comment:
If you drag a message on to the placeholder conversation, then create a new conversation.